internal/backends/pubmed/client.go

```diff
@@ -78,23 +96,38 @@ func (c *Client) GetPublication(id string) (*models.Publication, error) {
 	if res := attrs.Get("title"); res.Exists() {
 		p.Title = res.String()
 	}
-	if res := attrs.Get("journalTitle"); res.Exists() {
-		p.Publication = res.String()
-	}
-	if res := attrs.Get("journalVolume"); res.Exists() {
-		p.Volume = res.String()
-	}
-	if res := attrs.Get("journalIssn"); res.Exists() {
-		p.ISSN = reSplit.Split(res.String(), -1)
+	if journalInfo := attrs.Get("journalInfo"); journalInfo.IsObject() {
+		if journalVolume := journalInfo.Get("volume"); journalVolume.Exists() {
+			p.Volume = journalVolume.String()
+		}
+		if journalIssue := journalInfo.Get("issue"); journalIssue.Exists() {
+			p.Issue = journalIssue.String()
+		}
+		if journal := journalInfo.Get("journal"); journal.IsObject() {
+			if journalTitle := journal.Get("title"); journalTitle.Exists() {
+				p.Publication = journalTitle.String()
+			}
+			if journalISSN := journal.Get("issn"); journalISSN.Exists() {
+				p.ISSN = append(p.ISSN, journalISSN.String())
+			}
+			if journalEISSN := journal.Get("essn"); journalEISSN.Exists() {
+				p.EISSN = append(p.EISSN, journalEISSN.String())
+			}
+		}
 	}
 	if res := attrs.Get("pubYear"); res.Exists() {
 		p.Year = res.String()
 	}
-	if res := attrs.Get("authorString"); res.Exists() {
-		for _, r := range reSplit.Split(res.String(), -1) {
-			nameParts := strings.Split(strings.ReplaceAll(r, ".", ""), " ")
-			firstName := nameParts[len(nameParts)-1]
-			lastName := strings.Join(nameParts[:len(nameParts)-1], " ")
+	if authorList := attrs.Get("authorList.author"); authorList.IsArray() {
+		for _, author := range authorList.Array() {
+			var firstName string
```

Comment:
`firstName := author.Get("firstName").String()`

I think the other importers also add `[missing]` if a name part is empty (see crossref importer)

Reply:
done